### PR TITLE
fix: add tests for sortedSetFetchByScore, fix minor issue with 0 min/max scores

### PR DESCRIPTION
**Pull request by cprice404**
Adds tests for sortedSetFetchByScore.  Fixes a minor issue where if min/max score were explicitly set to 0, we weren't passing them through to the request because they evaluated as "false-y" when we checked to see if they had been provided.

### CODE REVIEW

File: test/integration/sorted-set.test.ts [hunk 1]
```diff
@@ -616,6 +616,487 @@ describe('Integration tests for sorted set operations', () => {
...
+          sortedSetName,
+          {
+            minScore: 2_000_000,
+          }
+        );
+
+        expect(response).toBeInstanceOf(CacheSortedSetFetch.Hit);
+        const hitResponse = response as CacheSortedSetFetch.Hit;
+        expect(hitResponse.valueArray()).toEqual([]);
+      });
```
Comment (cprice404) [inline test/integration/sorted-set.test.ts:781]: @wandaitzuchen - same as previous PR - I don't necessarily need you to review this, but there are 3 tests that I added in this PR that are asserting that we get back a Hit with an empty list of values, but we are actually getting back a Miss.

File: src/internal/cache-client.ts [hunk 2]
```diff
@@ -1967,7 +1968,7 @@ export class CacheClient {
     count?: number
   ): Promise<CacheSortedSetFetch.Response> {
     const by_score = new grpcCache._SortedSetFetchRequest._ByScore();
-    if (minScore) {
+    if (minScore !== undefined) {
```
Comment (malandis) [inline src/internal/cache-client.ts:1971]: This is because `0` evaluates to `false` in JS, correct?

File: test/integration/sorted-set.test.ts [hunk 3]
```diff
@@ -619,6 +619,496 @@ describe('Integration tests for sorted set operations', () => {
...
+        const hitResponse = response as CacheSortedSetFetch.Hit;
+        expect(hitResponse.valueArray()).toEqual([
+          {value: 'bam', score: 1000},
+          {value: 'burrito', score: 9000},
+          {value: 'taco', score: 90210},
+        ]);
+      });
+
+      // Skipping this for now; need to re-enable:
+      // https://github.com/momentohq/client-sdk-nodejs/issues/330
```
Comment (malandis) [inline test/integration/sorted-set.test.ts:932]: 🎗️

**Review by malandis — APPROVED** [review]
Looks good; just need the server side fix to update the tests.

**Review by malandis — APPROVED** [review]
